documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -21,14 +21,13 @@ endif::[]
 [id="about-mtv"]
 == About {the-lc} {project-full}
 
-You can migrate virtual machines from VMware vSphere, {rhv-full}, or {osp} source providers to {virt} with {the-lc} {project-first}.
+You can migrate virtual machines from VMware vSphere, Open Virtual Application (OVA) files that were created by or are compatible with VMware vSphere, {rhv-full}, {ocp-short}, or {osp} source providers to {virt} with {the-lc} {project-first}.
```

Comment:
Looking good, I'm just a bit unsure about the name, isn't it too long?
maybe we should keep it as is here but just use OVA in all rest of the places?